### DISCUSSION TITLE
Does changing Cobra/Disc-Less Payload affect entire system, or just Irisman?

### DISCUSSION THREAD

**Discussion by ghost** [aldostools/IRISMAN#33] [General]
I'd like to leave the option on "with BDVD controller," since I only use Irisman for disc dumping, but I'm not sure if doing that will affect Webman (or any other manager, in that regard).

**Comment by aldostools** [comment]
That option only affects IRISMAN. "With BDVD controller" basically tells the backup manager that don't use Cobra and that allow to read the bdvd disc for backup and other tasks.

You can leave it activated in IRISMAN, and use webMAN MOD to mount ISO and other game types.

**Comment by ghost** [comment]
Thanks for the info, Aldo! 
And thank you for all your work on Webman-MOD and irisman, they are great tools.
